gui/qt/main_window.py

```diff
@@ -1473,18 +1486,21 @@ def read_send_tab(self):
         coins = self.get_coins(isInvoice)
         return outputs, fee, label, coins
 
+    def do_preview_crowdfund(self):
+        self.do_send(preview = True, crowdfunding = True)
+
     def do_preview(self):
         self.do_send(preview = True)
 
-    def do_send(self, preview = False):
+    def do_send(self, preview = False, crowdfunding = False):
         if run_hook('abort_send', self):
             return
         r = self.read_send_tab()
         if not r:
             return
         outputs, fee, tx_desc, coins = r
-        try:
-            tx = self.wallet.make_unsigned_transaction(coins, outputs, self.config, fee)
+        try: 
+            tx = self.wallet.make_unsigned_transaction(coins, outputs, self.config, fee,None,crowdfunding)
```

Comment:
Man.. please follow coding conventions of the app -- we put spaces after commas (usually) unless we're being intentionally terse. Try and follow that because it breaks the flow to see stuff like this...

Reply:
I fixed it in a bunch of places but left this there so you can see what I mean..